packages/plugin-lighthouse/src/lib/lighthouse-plugin.integration.test.ts

```diff
@@ -1,12 +1,76 @@
-import { expect } from 'vitest';
-import { pluginConfigSchema } from '@code-pushup/models';
-import { lighthousePlugin } from './lighthouse-plugin';
+import { rm } from 'node:fs/promises';
+import { join } from 'node:path';
+import { afterEach, expect } from 'vitest';
+import { AuditOutput, pluginConfigSchema } from '@code-pushup/models';
+import {
+  getLogMessages,
+  shouldSkipLongRunningTests,
+} from '@code-pushup/test-utils';
+import { ui } from '@code-pushup/utils';
+import { createRunnerFunction, lighthousePlugin } from './lighthouse-plugin';
+
+const lighthousePluginTestFolder = join('tmp', 'plugin-lighthouse');
 
 describe('lighthousePlugin', () => {
   it('should create valid plugin config', () => {
-    const pluginConfig = lighthousePlugin('https://code-pushup-portal.com');
+    const pluginConfig = lighthousePlugin('https://www.google.com/');
     expect(() => pluginConfigSchema.parse(pluginConfig)).not.toThrow();
     expect(pluginConfig.audits.length).toBeGreaterThan(100);
     expect(pluginConfig.groups).toHaveLength(5);
   });
 });
+
+describe('createRunnerFunction', () => {
```

Comment:
🧪 Test maintainability: Both the creation and execution of the runner function is tested here.
```suggestion
describe('runner creation and execution', () => {
```